src/classes/PageList.php

```diff
@@ -19,7 +20,11 @@ public function get_origin() {
   }
 
   public function render() {
-    $template = Helpers::get_template_path('_page_list.php');
+     
+      $template = Helpers::get_template_path('_page_list.php');
```

Comment:
Would you mind fixing indentation (should be 2 spaces)?  Also could you remove the blank lines on L23, L25-L27?

Reply:
👍